examples/evaluation/promptiter_regression_loop/output/optimization_report.md

```
@@ -0,0 +1,117 @@
+
+- Gate: **rejected**
+- Candidate prompt:
+
+```text
+Rephrase without changing behavior. fixture:ineffective
+```
+  - validation score gain 0.000000 is below required 0.300000
+- Train score: 0.666667 (+0.000000)
+- Validation score: 0.666667 (+0.000000)
+- Evaluation cost: 12 calls, 60 tokens, 24 ms
+- Optimization cost: 22 calls, 142 tokens, 64 ms
```

markdownlint-cli2 (0.23.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)